slither/tools/read_storage/read_storage.py

```diff
@@ -0,0 +1,309 @@
+import sys
+import logging
+from math import floor
+
+from hexbytes import HexBytes
```

Comment:
A lot of dependencies in this file are not present in the setup.py (including hexbytes, all the eth_*)

Maybe we could group them into an import statement, like

```python
def load_dependencies():
   try:
    # Try to import every dep and
    # print a message if it's not installed


hexbytes, eth_utils,  eth_utils= load_dependencies()


```

Reply:
They're web3 dependencies so they'll be installed with it. I will follow this suggestion